src/pages/clients/statement/Statement.tsx

```diff
@@ -62,15 +66,20 @@ export function Statement() {
 
   const dates = [
     {
-      id: 'last_7_days',
+      id: 'last7_days',
```

Comment:
This part has been changed to match date keys on schedule page for API request. This change does not have impact on functionality on Statement page.